AutocompleteClient/FW/Logic/Result/CIOAutocompleteResult.swift

```diff
@@ -34,7 +37,10 @@ import Foundation
         
         self.data = json[Constants.Result.data] as? JSONObject ?? [:]
         
-        // TODO: Write map here
+        guard let id = data["id"] as? String else{
```

Comment:
Tests are not passing.  Getting some fairly simple issues with swapping `itemName` for `itemID` in conversion request builder tests, but also `Thread 1: EXC_BAD_INSTRUCTION (code=EXC_I386_INVOP, subcode=0x` on 
```
 func mockCIOResult(_ value: String, groupName: String? = nil) -> CIOResult {
        return CIOResult(autocompleteResult: CIOAutocompleteResult(json: ["value": value])!, group: nil)
    }
```
I think it has something to do with the id property.  Maybe id isn't nullable or something to that effect?

Reply:
Seems like mocking a result requires a couple of properties to be present. I took a single result JSON and extracted it to a separate file, so we can load the result from it. It seems like the safest solution atm.